src/app/modules/user/components/add-hours-choice/add-hours-choice.component.scss

```diff
@@ -0,0 +1,44 @@
+@import 'base/variables';
+@import 'base/fonts';
+
+.add-hours-choice {
+  &__title {
+    color: $dark_1;
+
```

Comment:
in the stylesheets / base / _fonts.scss there is a class ".text_dark_1" that you can use to place colors through a class in HTML. This is not necessary, but it would be good for a consistent coding style across the application.